src/core/components/operation-summary-path.jsx

```diff
@@ -37,7 +37,8 @@ export default class OperationSummaryPath extends PureComponent{
     const DeepLink = getComponent( "DeepLink" )
 
     return(
-      <span className={ deprecated ? "opblock-summary-path__deprecated" : "opblock-summary-path" } 
+      <span className={ deprecated ? "opblock-summary-path__deprecated" : "opblock-summary-path" }
+        id={`${tag}-${operationId}`}
```

Comment:
I'm assuming this ID will be unique, but as I mentioned in the PR summary it would be safer to include a uuid package to generate unique IDs. If that seems like a good idea, I would be happy to update this PR with that change

Reply:
### **Change request**: use `path + method` instead of tag + operationId here.

Combinations of tag and operationId are not technically guaranteed to be unique, although the OpenAPI spec says non-unique operationIds are illegal. We've had complaints from users in places where we track state by operationId (e.g. https://github.com/swagger-api/swagger-ui/issues/5096). You can also see this in action on the [LA Metro Arrivals API](https://api.metro.net/) - click on `/agencies/{agency_tag}/routes/`, notice that more than one operation expands (I've been meaning to send them some revisions to their document...).

Fortunately, because (a) paths and methods in OpenAPI documents are key names, and (b) YAML/JSON doesn't allow duplicated key names, `path + method` is guaranteed to be unique.

This is another instance of accessibility features not being safe for use cases that include multiple Swagger UI instances on one page though - see my thoughts on that in https://github.com/swagger-api/swagger-ui/pull/5299#discussion_r280961334.

Reply:
FYI: you should be able to pull `method` out of `operationProps.toJS()`.